packages/realm-web-integration-tests/src/user.test.ts

```diff
@@ -38,6 +38,14 @@ describe("User", () => {
         expect(user.customData).deep.equals({});
     });
 
+    it("can be stringified", async () => {
+        const app = createApp();
+        const credentials = Credentials.anonymous();
+        const user = await app.logIn(credentials);
+        const output = JSON.stringify(user);
+        expect(typeof output).equals("string");
```

Comment:
Here we could parse it back into an object & check that the required keys exists.